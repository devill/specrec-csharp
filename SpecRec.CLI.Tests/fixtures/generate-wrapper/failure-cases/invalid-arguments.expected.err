Required argument missing for command: 'generate-wrapper'.