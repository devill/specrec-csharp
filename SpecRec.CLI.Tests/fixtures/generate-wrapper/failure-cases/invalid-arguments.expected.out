Description:
  Generate wrapper class and interface for a given class

Usage:
  SpecRec.CLI generate-wrapper <className> [options]

Arguments:
  <className>  The name of the class to wrap

Options:
  -h, --hierarchy-mode <hierarchy-mode>  How to handle inheritance hierarchy: 'single' or 'full' [default: prompt]
  -?, -h, --help                         Show help and usage information